golang/cmd/mqtt-to-postgresql/main.go

```diff
@@ -11,19 +11,44 @@ import (
 	"syscall"
 	"time"
 
-	"github.com/beeker1121/goque"
 	"github.com/heptiolabs/healthcheck"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/united-manufacturing-hub/united-manufacturing-hub/internal"
 	"go.uber.org/zap"
 )
 
-var globalPQ *goque.PrefixQueue
+var addOrderHandler AddOrderHandler
+var addParentToChildHandler AddParentToChildHandler
+var addProductHandler AddProductHandler
+var addShiftHandler AddShiftHandler
+var countHandler CountHandler
+var deleteShiftByAssetIdAndBeginTimestampHandler DeleteShiftByAssetIdAndBeginTimestampHandler
+var deleteShiftByIdHandler DeleteShiftByIdHandler
+var endOrderHandler EndOrderHandler
+var maintenanceActivityHandler MaintenanceActivityHandler
+var modifyProducedPieceHandler ModifyProducedPieceHandler
+var modifyStateHandler ModifyStateHandler
+var productTagHandler ProductTagHandler
+var recommendationDataHandler RecommendationDataHandler
+var scrapCountHandler ScrapCountHandler
+var scrapUniqueProductHandler ScrapUniqueProductHandler
+var startOrderHandler StartOrderHandler
+var productTagStringHandler ProductTagStringHandler
+var stateHandler StateHandler
+var uniqueProductHandler UniqueProductHandler
+var valueDataHandler ValueDataHandler
+var valueStringHandler ValueStringHandler
+var storedRawMQTTHandler StoredRawMQTTHandler
 
 func main() {
-
 	// Setup logger and set as global
-	logger, _ := zap.NewDevelopment()
+	var logger *zap.Logger
+	if os.Getenv("LOGGING_LEVEL") == "DEBUG" {
```

Comment:
I think not consistent with the other Logging Level (DEVELOPMENT). Please create issue to standardize environment variables, e.g. LOGGING_LEVEL